GenderPayGap.WebUI/Views/Components/CovidMessage.cshtml

```diff
@@ -1,11 +1,10 @@
 <div class="covid-message">
-    <p class="govuk-body govuk-!-font-weight-bold">
-        Coronavirus (COVID-19): There have been some important changes to enforcement of the
-        gender pay gap reporting regulations this year due to the impact of COVID-19 on employers.
-        For more information please read the 
+    <p class="govuk-body govuk-!-font-weight-bold govuk-!-margin-bottom-0">
+        There were some important changes to enforcement of the gender pay gap reporting regulations for the 2019/20 
+        reporting year which used a snapshot date of either 31 March 2019 for most public authority employers, 
+        or 5 April 2019 for private, voluntary, and all other public authority employers. For more information please read 
         <a href="https://www.gov.uk/government/news/employers-do-not-have-to-report-gender-pay-gaps" class="govuk-link" rel="noopener">
-            press release from the Minister for Women and
-            Equalities and David Isaac, Chair of the Equality and Human Rights Commission.
+            this press release.
```

Comment:
This new wording is quite confusing. In particular, the bit about snapshot dates.
I think we should go back to the team and suggest a clearer wording.

Problems:
* This message appears on public-facing pages, but members of the public don't know (and shouldn't need to know) what a snapshot date is
* The message reads like the information about snapshot dates is the main information we're conveying (it sounds like we're telling people that snapshot dates have changed)

Suggestion:
> There were some important changes to enforcement of the gender pay gap reporting regulations for the 2019/20 reporting year. For more information please read *this press release*

Alternative suggestions (if we _must_ mention snapshot dates):
(uses brackets and removes information about why you'd choose one snapshot date or another - since this isn't relevant in this context)
> There were some important changes to enforcement of the gender pay gap reporting regulations for the 2019/20 reporting year (which used a snapshot date of either 31 March 2019 or 5 April 2019). For more information please read *this press release*

or
(at least uses brackets)
> There were some important changes to enforcement of the gender pay gap reporting regulations for the 2019/20 reporting year (which used a snapshot date of either 31 March 2019 for most public authority employers, or 5 April 2019 for private, voluntary, and all other public authority employers). For more information please read *this press release*


Notes:
* The press release itself doesn't mention snapshot dates, so the press office must think it's clear enough just using the reporting year